deploy/helm/secret-operator/templates/daemonset.yaml

```diff
@@ -69,7 +69,7 @@ spec:
       volumes:
         - name: registration-sock
           hostPath:
-            path: /var/lib/kubelet/plugins_registry/secrets.stackable.tech-reg.sock
+            path: /var/lib/kubelet/plugins_registry
```

Comment:
I don't know much about this but...did it point to a directory before as well?

And is there any chance of a namespace clash?

And...does YAML support comments? If yes...this'd be a great place for a link to the issue...

Reply:
The node-registrar is hard-coded to use `$REGISTRATION_SOCK_PATH/$CSI_DRIVER-reg.sock`. This lead to the actual path being `/var/lib/kubelet/plugins_registry/secrets.stackable.tech-reg.sock/secrets.stackable.tech-reg.sock` rather than the correct `/var/lib/kubelet/plugins_registry/secrets.stackable.tech-reg.sock`.

In most cases it still worked fine, because kubelet scans the plugin registry path recursively... except for that it can't connect when the path gets too long (#229).

I can add a comment about it, sure.

Reply:
Done.

Reply:
Sorry for being pedantic and I do know git blame exists but I find it incredibly helpful if code links to issues

```
# node-driver-registrar appends a driver-unique filename to this path to avoid conflicts
# see https://github.com/stackabletech/secret-operator/issues/229 for why this path should not be too long
```

Reply:
Good point.